grakn-dist/src/grakn

```diff
@@ -0,0 +1,662 @@
+      grakn_wait_until_ready
+      status=$?
+      echo "SUCCESS"
+    else
+      # exec grakn bin failed
+      status=1
+      echo "FAILED!"
+    fi
+
+    return $status
+  fi
+}
+
+grakn_server_stop_grakn() {
+  echo -n "Stopping Grakn..."
+  grakn_is_grakn_initiated
+  local is_grakn_running=$?
+  if [[ $is_grakn_running -eq 0 ]]; then
+    if [[ -e "$GRAKN_PID" ]]; then
+      kill `cat $GRAKN_PID`
+      status_kill=$?
+      rm $GRAKN_PID
+      if [ $status_kill -eq 0 ] ; then
+        echo "SUCCESS"
+      else
+        echo "FAILED!" # unable to kill
+      fi
+      return $status_kill
+    else
+      echo "FAILED!" # unable to find PID file
+      return 1
+    fi
+  else
+    echo "NOT RUNNING"
+  fi
+}
+
+grakn_clean_grakn() {
+  # redirect to /dev/null to silence 'No such file' error, when the directory is already empty
+  rm "${GRAKN_HOME}"/logs/*.log > /dev/null 2>&1
+}
+
+# ================================================
+# command line helpers
+# ================================================
+grakn_server_stop() {
+  grakn_server_stop_grakn
+  status_grakn=$?
+
+  grakn_server_stop_queue
+  status_queue=$?
+
+  grakn_server_stop_storage
+  status_storage=$?
+
+  if [ $status_grakn -eq 0 ] && [ $status_queue -eq 0 ] && [ $status_storage -eq 0 ] ; then
+    return 0
+  else
+    return 1
+  fi
+}
+
+# attempt to start grakn components. if one of them fails, abort everthing
+grakn_server_start_grakn() {
+  grakn_start_and_wait_until_ready
+  grakn=$?
+  # log
+  if [ $grakn -ne 0 ] ; then
+    echo "Unable to start Grakn"
+  fi
+  return $grakn
+}
+
+grakn_server_start_queue() {
+  queue_start_and_wait_until_ready
+  queue=$?
+  # log
+  if [ $queue -ne 0 ] ; then
+    echo "Unable to start Queue"
+  fi
+  return $queue
+}
+
+grakn_server_start_storage() {
+  storage_start_and_wait_until_ready
+  storage=$?
+  # log
+  if [ $storage -ne 0 ] ; then
+    echo "Unable to start Storage"
+  fi
+  return $storage
+}
+
+grakn_server_start() {
+  grakn_server_start_storage
+  local storage=$?
+
+  grakn_server_start_queue
+  local queue=$?
+
+  grakn_server_start_grakn
+  local grakn=$?
+
+  local status=
+  if [ $storage -ne 0 ] || [ $queue -ne 0 ] || [ $grakn -ne 0 ]; then
+    status=1
+  else
+    status=0
+  fi
+
+  return $status
+}
+
+grakn_server_help() {
+  echo "Usage: grakn server COMMAND
+
+COMMAND:
+start [grakn|queue|storage]  Start Grakn (or optionally, only one of the component)
+stop [grakn|queue|storage]   Stop Grakn (or optionally, only one of the component)
+status                         Check if Grakn is running
+clean                          DANGEROUS: wipe data completely
+
+Tips:
+- Start Grakn with 'grakn server start'
+- Start or stop only one component with, e.g. 'grakn server start storage' or 'grakn server stop storage', respectively
+"
+}
+grakn_server_status() {
+  storage_is_storage_running
+  if [ $? -eq 0 ] ; then
+    echo "Casssandra: RUNNING"
+  else
+    echo "Casssandra: NOT RUNNING"
```

Comment:
"Storage: NOT RUNNING"

Reply:
fixed in 0d4ff9888fa1efb245effe555cfd34ffb161c3c3